packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -389,9 +389,15 @@ export class AppStudioPlugin implements Plugin {
       return await this.buildTeamsPackage(ctx);
     } else if (func.method === "getAppDefinitionAndUpdate") {
       if (func.params && func.params.type && func.params.manifest) {
+        let isLocalDebug;
```

Comment:
nit: seems the logic can be simplified like `const isLocalDebug = (func.params.type as string) === "localDebug"`